src/models/converters/transactions/mod.rs

```diff
@@ -191,12 +191,19 @@ impl MultisigTransaction {
     }
 
     fn to_custom(&self, info_provider: &mut dyn InfoProvider) -> Custom {
+        let data_size = data_size(&self.data).to_string();
+        let value = self.value.as_ref().unwrap().to_owned();
+        let method_name = self.data_decoded.as_ref().map(|it| it.method.to_owned());
         Custom {
             to: self.to.to_owned(),
             to_info: info_provider.address_info(&self.to).ok(),
-            data_size: data_size(&self.data).to_string(),
-            value: self.value.as_ref().unwrap().into(),
-            method_name: self.data_decoded.as_ref().map(|it| it.method.to_owned()),
+            is_cancellation: &self.to == &self.safe
```

Comment:
We need to check all params of thebsafe tx. See current implementation from web linked in the issue